tests/test_return_value_policy_override.cpp

```diff
@@ -1,11 +1,69 @@
+#include <pybind11/smart_holder.h>
+
 #include "pybind11_tests.h"
 
 namespace test_return_value_policy_override {
 
 struct some_type {};
 
+struct obj {
+    std::string mtxt;
+    obj(const std::string &mtxt_) : mtxt(mtxt_) {}
+    obj(const obj &other) { mtxt = other.mtxt + "_CpCtor"; }
+    obj(obj &&other) { mtxt = other.mtxt + "_MvCtor"; }
+    obj &operator=(const obj &other) {
+        mtxt = other.mtxt + "_CpCtor";
+        return *this;
+    }
+    obj &operator=(obj &&other) {
+        mtxt = other.mtxt + "_MvCtor";
+        return *this;
+    }
+};
+
+struct nocopy {
+    std::string mtxt;
+    nocopy(const std::string &mtxt_) : mtxt(mtxt_) {}
```

Comment:
Don't you still need to delete the copy ctor?

Reply:
Thanks for pointing this out. I forgot this. I deleted the copy ctor.